pages/rules/index.mdx

```diff
@@ -0,0 +1,28 @@
+import Head from "next/head";
+import Link from "next/link";
+import styles from "./rules.module.scss";
+import Page from "@layouts/page/page";
+import Header from "@components/header/header";
+import Footer from "@components/footer/footer";
+import { HL, H2, Text } from "@components/typography/typography";
+import Stack from "@components/stack/stack";
+import Rules from "./rules.md";
+
+<div>
+  <Head>
+    <title>Rules - UNIHACK</title>
+  </Head>
+  <Page>
+    <Header />
+    <section className={styles["rules"]}>
+      <div className={styles["rules-content"]}>
+        <Stack size="medium">
+          <HL>Rules</HL>
+          <Text>
+            <Rules />
```

Comment:
Don't put this this inside `<Text>` because this would wrap all the HTML elements in this markdown file inside another paragraph element. `<Rules />` should be just fine by itself under the heading.